core/src/main/scala/com/metamx/tranquility/druid/DruidRollup.scala

```diff
@@ -35,7 +35,8 @@ import scala.collection.JavaConverters._
 class DruidRollup(
   val dimensions: DruidDimensions,
   val aggregators: IndexedSeq[AggregatorFactory],
-  val indexGranularity: QueryGranularity
+  val indexGranularity: QueryGranularity,
+  val isRollup: Boolean
```

Comment:
Can you give this a default value of `true` for backwards compatibility with existing source? (people won't have the extra arg defined)